invisible_cities/reco/detsim_functions.py

```diff
@@ -0,0 +1,26 @@
+import numpy  as np
+import tables as tb
+import pandas as pd
+
+from typing    import   List, Tuple, Callable, Generator
+
+from invisible_cities.io.mcinfo_io import read_mchits_df
+from invisible_cities.reco.corrections_new import read_maps
+
+
+#######################################
+############### SOURCE ################
+#######################################
+def load_MC(files_in : List[str]) -> Generator:
+    for filename in files_in:
+        with tb.open_file(filename) as h5in:
+            extents = pd.read_hdf(filename, 'MC/extents')
```

Comment:
This will be affected (simplified really) by the changes @paolafer and @jmunozv are making to the nexus persistency. The timescale looks like we'll be better waiting to approve this after the adjustments.

For now I'd change the two lines to one since you only read here for the event list so:
`event_ids = pd.read_hdf(filename, 'MC/extents).evt_number`